practitioner-guides/introduction.md

```diff
@@ -59,6 +59,7 @@ You should also continue to monitor it over the long-term to see if your improve
 # Additional Reading
 
 * [Practitioner Guide series](https://chaoss.community/about-chaoss-practitioner-guides/) with guides to help you improve responsiveness, contributor sustainability, organizational participation, and more.
+* A short video about each guide can be found in the [Practitioner Guides Playlist](https://www.youtube.com/watch?v=xTK2xsd4jZE&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z) in the CHAOSS YouTube channel.
```

Comment:
I think this should be https://www.youtube.com/playlist?list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z